### PR TITLE
Replace `Change Password` with `Dashboard` on header #38 #93

### PR DESCRIPTION
**Pull request by SwarupKharul**

### CODE REVIEW

**Review by SwarupKharul — COMMENTED** [review]
Brought home link above signup

File: templates/snippets/header.html [hunk 1]
```diff
@@ -10,23 +10,24 @@ <h2 class="mb-3 my-md-0 mr-md-auto font-weight-normal" >
...
     {% endif %}
 </div>
 {% endcomment %}
 
 <nav class="navbar navbar-expand-sm bg-dark navbar-dark sticky-top p-3 px-md-4">
-    <h2 class="my-0 mr-md-auto font-weight-normal" >
-        <a class="text-decoration-none" href="{% url 'home:index' %}" style="color:#ff8c00;">Jagrati</a>								<!--Change this-->
+    <h2 class="my-0 mr-md-auto font-weight-normal">
+        <a class="text-decoration-none" href="{% url 'home:index' %}" style="color:#ff8c00;">Jagrati</a>
+        <!--Change this-->
```
Comment (garg3133) [inline templates/snippets/header.html:30]: Remove this line.

File: templates/snippets/header.html [hunk 2]
```diff
@@ -49,25 +50,30 @@ <h2 class="my-0 mr-md-auto font-weight-normal" >
...
             <li class="nav-item">
                 <a class="btn btn-outline-danger" href="{% url 'accounts:logout' %}">Sign Out</a>
             </li>
             {% endif %}
-            
+
             {% else %}
+             <li class="nav-item">
+                <a class="btn btn-outline-danger" href="{% url 'home:index' %}">Home</a>
+            </li>
```
Comment (garg3133) [inline templates/snippets/header.html:72]: This button should be like `Dashboard` button in appearance and not like `Sign In` button. You can check the appearance of this button at http://localhost:8000/accounts/password_reset/done/

**Review by garg3133 — COMMENTED** [review]
You should never make any changes and open a PR from `master` branch. Before making any changes, always create a new branch first and then make your changes.

Also, can you squash all your commits into one commit? Follow the below steps:
* `git rebase -i HEAD~<no. of commits you made in total>`
* Except for first `pick`, replace all `pick` with `squash` in the file opened.
* Close the file.
* Force push your changes. (`git push origin <your-brach-name>`).

File: templates/snippets/header.html (hunk 1 above)
Comment (SwarupKharul) [reply templates/snippets/header.html:30]: can u please specify line number.

Comment (garg3133) [reply templates/snippets/header.html:30]: Line 30 (Change this comment).

**Comment by garg3133** [comment]
Resolved in #96.

**Pull request closed without merging**